returnn/tf/layers/basic.py

```diff
@@ -2627,6 +2627,12 @@ def get_out_data_from_opts(cls, name, axes, sources=(), n_out=NotSpecified, out_
         # Time got merged with feature or batch.
         data.time_dim_axis = None
     data.feature_dim_axis = new_feature_dim_axis
+    # Temporarily set size_placeholder, dyn_size values will be set correctly in __init__
```

Comment:
The comment is a bit confusing. This is often not temporarily.
You might say that it would be updated for merged axes in `__init__` or so.

Reply:
Okay, jep sure!